ui/js/tree_url_app.js

```diff
@@ -0,0 +1,4 @@
+
+
+var app = angular.module("tree_url_app", []);
```

Comment:
this is the part that Ed was saying is not needed.  We don't need to create a new app, we should add to the existing ``treeherderApp``.

I suggest adding your functions to this factory: https://github.com/mozilla/treeherder/blob/master/ui/js/services/main.js#L13-24

Then you can reference that factory in any of the controllers that need to use it.